src/main/java/com/minecolonies/coremod/colony/managers/CitizenManager.java

```diff
@@ -457,15 +463,38 @@ public void onWorldTick(final TickEvent.WorldTickEvent event)
             getCitizens().stream().filter(Objects::nonNull).forEach(CitizenData::updateCitizenEntityIfNecessary);
         }
 
-        //  Spawn Citizens
-        if (colony.hasTownHall() && getCitizens().size() < getMaxCitizens())
+        //  Spawn initial 4 Citizens
+        if (colony.hasTownHall() && getCitizens().size() < 4)
         {
             int respawnInterval = Configurations.gameplay.citizenRespawnInterval * TICKS_SECOND;
             respawnInterval -= (SECONDS_A_MINUTE * colony.getBuildingManager().getTownHall().getBuildingLevel());
 
             if ((event.world.getTotalWorldTime() + 1) % (respawnInterval + 1) == 0)
             {
-                spawnCitizen();
+                // Make sure the initial citizen contain both genders
+                CitizenData newCitizen = createAndRegisterNewCitizenData();
+
+                boolean femaleExists = false;
+
+                for (CitizenData data : citizens.values())
+                {
+                    if (data.isFemale())
+                    {
+                        femaleExists = true;
+                        break;
+                    }
+                }
+
+                if (!femaleExists)
+                {
+                    newCitizen.setIsFemale(true);
+                }
+                else
+                {
+                    newCitizen.setIsFemale(false);
```

Comment:
so we have only 1 woman always?

Reply:
atm yes, but could change it to 50-50 instead if you want